public/robots.txt

```diff
@@ -1,3 +1,2 @@
 User-Agent: *
-Disallow: /search/
```

Comment:
Apparently, this `robots.txt` isn't actually being served. Either way, this is fine.